python/lsst/ts/standardscripts/auxtel/focus_sweep_latiss.py

```diff
@@ -0,0 +1,147 @@
+        return self.latiss
+
+    async def configure_tcs(self) -> None:
+        """Handle creating the ATCS object and waiting remote to start."""
+        if self.atcs is None:
+            self.log.debug("Creating ATCS.")
+            self.atcs = ATCS(
+                domain=self.domain,
+                log=self.log,
+            )
+            await self.atcs.start_task
+        else:
+            self.log.debug("ATCS already defined, skipping.")
+
+    async def configure_camera(self) -> None:
+        """Handle creating the camera object and waiting remote to start."""
+        if self.latiss is None:
+            self.log.debug("Creating Camera.")
+            self.latiss = LATISS(
+                self.domain, intended_usage=LATISSUsages.TakeImage, log=self.log
+            )
+            await self.latiss.start_task
+        else:
+            self.log.debug("Camera already defined, skipping.")
+
+    @classmethod
+    def get_schema(cls) -> dict:
+        schema_dict = super().get_schema()
+        additional_properties = yaml.safe_load(
+            """
+            filter:
+                description: Filter name or ID; if omitted the filter is not changed.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: null
+            grating:
+                description: Grating name; if omitted the grating is not changed.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: null
+        """
+        )
+        schema_dict["properties"].update(additional_properties)
+        return schema_dict
+
+    def get_instrument_configuration(self) -> dict:
+        return dict(
+            filter=self.config.filter,
+            grating=self.config.grating,
+        )
+
+    def get_instrument_filter(self) -> str:
+        """Get instrument filter configuration.
+
+        Returns
+        -------
+        instrument_filter: `string`
+        """
+        filter = self.config.filter
+        grating = self.config.grating
+        return f"{filter}~{grating}"
+
+    def get_instrument_name(self) -> str:
+        """Get instrument name.
+
+        Returns
+        -------
+        instrument_name: `string`
+        """
+        return self.instrument_name
+
+    async def move_hexapod(self, axis: str, value: float) -> None:
+        """Move hexapod to a position along an axis for AuxTel."""
+
+        offset_args = {"x": 0, "y": 0, "z": 0, "u": 0, "v": 0}
+        offset_args[axis] = value
+        await self.atcs.offset_aos_lut(
+            x=offset_args["x"],
+            y=offset_args["y"],
+            z=offset_args["z"],
+            u=offset_args["u"],
+            v=offset_args["v"],
+        )
+
+    async def configure(self, config: types.SimpleNamespace) -> None:
```

Comment:
I think you can remove this one here. It is already defined in the parent class.